src/components/panel/examples/branded/index.njk

```diff
@@ -0,0 +1,11 @@
+---
+"theme": census
+---
+{% from "components/panel/_macro.njk" import onsPanel %}
+{{ 
+    onsPanel({
```

Comment:
Might be better to use the {% call onsPanel %} method for this one, as your body has a fair bit of html in there.